Sofa/Component/Topology/Container/Constant/src/sofa/component/topology/container/constant/MeshTopology.cpp

```diff
@@ -533,10 +533,13 @@ void MeshTopology::init()
 {
 
     BaseMeshTopology::init();
+
+    const auto& hexahedra = seqHexahedra.getValue();
+
     if (nbPoints==0)
     {
         // looking for upper topology
-        if (!seqHexahedra.getValue().empty())
+        if (!hexahedra.empty())
             m_upperElementType = core::topology::TopologyElementType::HEXAHEDRON;
         else if (!seqTetrahedra.getValue().empty())
```

Comment:
I saw in the code that it is the same .getValue() party with seqEdges, seqTrianges, seqTetra....
Will you do it as well ?

Reply:
It's a good point. It's done.